SpanishFilter/sections/antiadblock.txt

```diff
@@ -11,6 +11,9 @@ tunovelaligera.com#@#iframe[width="728"][height="90"]
 tunovelaligera.com#@#img[width="728"][height="90"]
 !#endif
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/73208
+! https://github.com/AdguardTeam/AdguardFilters/issues/73194
+centraldopirata.com,torrentsmegafilmes.com#%#//scriptlet("prevent-bab")
```

Comment:
Wrong filter. We have rules for BlockAdBlock in AdGuard Base. You have already done the similar case.
1. Add `centraldopirata.com,torrentsmegafilmes.com` to the common scriptlet rule.
2. The same for `! Old BAB (`$$script[wildcard="*eval*babasbmsgx*"][min-length="5000"][max-length="20000"]`)`
3. Add them to the `content_blocker` section
```
@@||centraldopirata.com^$generichide
@@||torrentsmegafilmes.com^$generichide
```

Reply:
Please check the new PR.